text/0038-output-types-for-tokenization-and-sc/0038-output-types-for-tokenization-and-sc.md

```diff
@@ -0,0 +1,2886 @@
+        </td>
+    </tr>
+    <tr>
+        <td valign="top">Sender <code>oneOf</code></td>
+        <td colspan="2">
+            <details>
+                <summary>Ed25519 Address</summary>
+                <table>
+                    <tr>
+                        <td><b>Name</b></td>
+                        <td><b>Type</b></td>
+                        <td><b>Description</b></td>
+                    </tr>
+                    <tr>
+                        <td>Address Type</td>
+                        <td>uint8</td>
+                        <td>
+                            Set to <strong>value 0</strong> to denote an <i>Ed25519 Address</i>.
+                        </td>
+                    </tr>
+                    <tr>
+                        <td>Address</td>
+                        <td>ByteArray[32]</td>
+                        <td>The raw bytes of the Ed25519 address which is a BLAKE2b-256 hash of the Ed25519 public key.</td>
+                    </tr>
+                </table>
+            </details>
+            <details>
+                <summary>Alias Address</summary>
+                <table>
+                    <tr>
+                        <td><b>Name</b></td>
+                        <td><b>Type</b></td>
+                        <td><b>Description</b></td>
+                    </tr>
+                    <tr>
+                        <td>Address Type</td>
+                        <td>uint8</td>
+                        <td>
+                            Set to <strong>value 8</strong> to denote an <i>Alias Address</i>.
+                        </td>
+                    </tr>
+                    <tr>
+                        <td>Address</td>
+                        <td>ByteArray[20]</td>
+                        <td>The raw bytes of the alias address which is a BLAKE2b-160 hash of the outputID that created it.</td>
+                    </tr>
+                </table>
+            </details>
+            <details>
+                <summary>NFT Address</summary>
+                <table>
+                    <tr>
+                        <td><b>Name</b></td>
+                        <td><b>Type</b></td>
+                        <td><b>Description</b></td>
+                    </tr>
+                    <tr>
+                        <td>Address Type</td>
+                        <td>uint8</td>
+                        <td>
+                            Set to <strong>value 16</strong> to denote an <i>NFT Address</i>.
+                        </td>
+                    </tr>
+                    <tr>
+                        <td>Address</td>
+                        <td>ByteArray[20]</td>
+                        <td>The raw bytes of the nft address which is a BLAKE2b-160 hash of the outputID that created it.</td>
+                    </tr>
+                </table>
+            </details>
+        </td>
+    </tr>
+</table>
+
+| :information_source:  Good to know about address format |
```

Comment:
Maybe I'm being a bit pedantic now but I think it would be clearer to have an Address section and not wait until we first encounter addresses to explain this. Just a suggestion :)